config/global.php

```diff
@@ -75,6 +75,18 @@
 
     'Piwik\EventDispatcher' => DI\object()->constructorParameter('observers', DI\get('observers.global')),
 
+    'login.whitelist.ips' => function (ContainerInterface $c) {
```

Comment:
Config is wrapped in DI so other plugins can add further IPs dynamically without the risk of changing the config when manipulating the config directly in DI